### PR TITLE
support custom args in agentmain for dynamic installation and uninstallation

### LINKED ISSUE #1552: The attach command supports user-defined parameters.
### What would you like to be enhanced?

I'm using attach to start the agent and plugin, and I want to pass parameters to my plugin on the command line instead of reading them from the config file

### Why is this needed?

Some plug-ins implement simple functions and do not need to use dynamic configuration and read parameters from the configuration file,Therefore, reading parameters directly from the command line can save costs.

### PR DESCRIPTION
**Pull request by lilai23**
**What type of PR is this?**

Enhancement

**What this PR does / why we need it?**

When use agentmain to install sermant-agent and plugins，now we can add additional arguments for custom requirements. This makes developers input some variable parameters，and get them in sermant

**Which issue(s) this PR fixes？**

Fixes #1552

**Does this PR introduce a user-facing change?**

You can add some arguments in command like : command=INSTALL-PLUGINS:monitor, city=hangzhou, mood=happy
(https://sermant.io/zh/document/user-guide/sermant-agent.html#%E5%8A%A8%E6%80%81%E5%AE%89%E8%A3%85%E6%8F%92%E4%BB%B6)

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/sermant-io/Sermant/issues) related with this PR before you start working on it.
- [x] Make sure you have squashed your change to one single commit.
- [x] GitHub Actions works fine in this PR.

### CODE REVIEW

File: sermant-agentcore/sermant-agentcore-premain/src/test/java/io/sermant/premain/common/BootArgsBuilderTest.java [hunk 1]
```diff
@@ -114,10 +114,11 @@ private static void setEnv(Map envMap) throws NoSuchFieldException, IllegalAcces
...
     public void testParseArgs() {
         String agentArgs = "appName=test,command=INSTALL_PLUGIN:monitor/flowcontrol,server.port=9000";
-        Map<String, Object> argsMap = AgentArgsResolver.resolveAgentArgs(agentArgs);
-        BootArgsBuilder.build(argsMap, PathDeclarer.getAgentPath());
-        Assert.assertEquals("test", argsMap.get("appName"));
-        Assert.assertEquals("9000", argsMap.get("server.port"));
-        Assert.assertEquals("INSTALL_PLUGIN:monitor/flowcontrol", argsMap.get("command"));
+        Map<String, String> agentArgsMap = AgentArgsResolver.resolveAgentArgs(agentArgs);
+        Map<String, Object> bootArgsMap = new HashMap<>(agentArgsMap);
+        BootArgsBuilder.build(bootArgsMap, PathDeclarer.getAgentPath());
```
Comment (hanbingleixue) [inline sermant-agentcore/sermant-agentcore-premain/src/test/java/io/sermant/premain/common/BootArgsBuilderTest.java:119]: Does bootArgsMap do anything here?

**Review by hanbingleixue — DISMISSED** [review]

File: sermant-agentcore/sermant-agentcore-premain/src/test/java/io/sermant/premain/common/BootArgsBuilderTest.java (hunk 1 above)
Comment (lilai23) [reply sermant-agentcore/sermant-agentcore-premain/src/test/java/io/sermant/premain/common/BootArgsBuilderTest.java:119]: Resolved. Assert should aim to bootArgsMap

File: sermant-agentcore/sermant-agentcore-core/src/main/java/io/sermant/core/command/CommandProcessor.java [hunk 2]
```diff
@@ -74,6 +77,7 @@ public static void process(String command) {
             return;
         }
         String commandArgs = commandInfo.length > 1 ? commandInfo[1] : null;
+        DynamicAgentArgsManager.getAgentArgs().putAll(agentArgsMap);
```
Comment (daizhenyu) [inline sermant-agentcore/sermant-agentcore-core/src/main/java/io/sermant/core/command/CommandProcessor.java]: putall methon should be implemented within DynamicAgentArgsManager

Comment (lilai23) [reply sermant-agentcore/sermant-agentcore-core/src/main/java/io/sermant/core/command/CommandProcessor.java]: done

**Review by daizhenyu — APPROVED** [review]

**Review by hanbingleixue — APPROVED** [review]

**Pull request merged**